config/glossary.json

```diff
@@ -29,6 +34,11 @@
     "definition": "The unique identifier of the audience for an issued token, identified within a JSON Web Token as the `aud` claim. The audience value is either the application (`Client ID`) for an ID Token or the API that is being called (`API Identifier`) for an Access Token. At Auth0, the Audience value sent in a request for an Access Token dictates whether that token is returned in an opaque or JWT format.",
     "short": "A unique identifier, <b>aud</b> in a token, for an application (ID Token) or an API (Access Token)."
   },
+  "auth0-dashboard": {
+    "title": "Auth0 Dashboard" ,
+    "definition": "Auth0’s main product in which you can register your application or API, connect to a user store or another identity provider, and configure your Auth0 services.",
```

Comment:
Main product is confusing to me for some reason. Is "primary user interface" another way to describe the dashboard?

Reply:
This one has been addressed. Thanks for the help!